menu/menu.yaml

```diff
@@ -494,6 +494,8 @@
     Articles:
     - Url: nservicebus/dependency-injection/child-lifetime
       Title: Child Lifetimes
+    - Url: nservicebus/dependency-injection/extensions-dependencyinjection
+      Title: Microsoft.Extensions.DependencyInjection
```

Comment:
thoughts about the menu name? I've gone with the Microsoft prefix as that seems most likely what users would be looking for?